pipeline-marketplace-images/github-sign-off/create-and-wait-for-approval

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+source create-issue
+source wait-approval
+
+# Run create_issue and safely capture its output and exit code
+set +e
+issue=$(create_issue)
+exit_code=$?
+set -e
+
+# 2 = skip-approval or auto-approve => exit cleanly
```

Comment:
Can we variablise this in the other scrip to? Feels a bit magic number-y when it is comments 💭